src/afrolabs/components/kafka.clj

```diff
@@ -1368,13 +1458,17 @@
 (s/def ::ktable-id (s/and string?
                           #(pos-int? (count %))))
 
+(s/def ::caught-up-once? boolean?)
 (s/def ::ktable-cfg (s/and ::clientless-consumer
-                           (s/keys :req-un [::ktable-id])))
+                           (s/keys :req-un [::ktable-id]
+                                   :opt-un [::caught-up-once?])))
```

Comment:
define `::caught-up-once?` flag for ktable configuration, which will optionally use the `CaughtUpOnceNotification` strategy for "completing" the initial load.